2024/political-donations-compare/main.js

```diff
@@ -0,0 +1,160 @@
+    32418, 32557, 32906, 32973, 33039, 33113, 33400, 33416, 34476, 34598, 45189,
+    45285, 45599, 47052, 47260, 47689, 48035, 49131, 49299, 49415, 49899, 50915,
+    50957, 51297, 51562, 51710, 52278, 53589, 53649, 53700, 53765, 53886, 53991,
+    54022, 54070, 54581, 54872, 54918, 55074, 55231, 56391, 57433, 57917, 58375,
+    58583, 59237, 59698, 62556, 63021, 63082, 64191, 64448, 64604, 64864, 65171,
+    65251, 65396, 65708, 65995, 66318, 66331, 66334, 66504, 66644, 67178, 67990,
+    68058, 68431, 68471, 68491, 68843, 70462, 70482, 70642, 71378, 71565, 72743,
+    73228, 76995, 76995, 77045, 77599, 77649, 77649, 77750, 79421, 80930, 81071,
+    81071, 81071, 81600, 81600, 81600, 81640, 81645, 82063, 82063, 82073, 82080,
+    82080, 82483, 82528, 82553, 82593, 82699, 82771, 82791, 82816, 83098, 83395,
+    83395, 83430, 83580, 84181, 84181, 84697, 84823, 84833, 84973, 85098, 85098,
+    85599, 85639, 85679, 85730, 85745, 85839, 85839, 85854, 85884, 85884, 85909,
+    86927, 86927, 86957, 87062, 87142, 87157, 87506, 87513, 87774, 88274, 88319,
+    88374, 88424, 88469, 88608, 88733, 88733, 88813, 89553, 89553, 89553, 89553,
+    89578, 89578, 89581, 89646, 89646, 91146, 91920, 95219, 95494, 96508, 97573,
+    97853, 97933, 98033, 98093, 98193, 99604, 99604, 99639, 99639, 99649, 99674,
+    99714, 99714, 99724, 99854, 99979, 99979, 99979, 99979, 100004, 100024,
+    100024, 100069, 100069, 100069, 100069, 100069, 100104, 100119, 100129,
+    100129, 100209, 100209, 100234, 100619, 100659, 100741
+];
+
+const xAxisLabels = [0, 100, 200];
+
+Highcharts.setOptions({
+    lang: {
+        thousandsSep: ','
+    }
+});
+
+Highcharts.chart('chart-div', {
+    // Setting default colors
+    colors: ['#800000', '#FFA319'],
+
+    // All code for your chart goes here
+    title: {
+        text: '2020 and 2024 Presidential Election Donations By UChicago Professors',
+        align: 'center'
+    },
+    subtitle: {
+        text: '2024 Election donations are matching the pace of the 2020 Election.',
+        align: 'center',
+        style: {
+            fontSize: '15px'
+        }
+    },
+    yAxis: {
+        title: {
+            text: 'Cumulative Donations'
+        },
+        labels: {
+            format: '${value:,.0f}'
+        },
+        tickInterval: 50000
+    },
+    xAxis: {
+        title: {
+            text: 'Days Until Election'
+        },
+        reversed: true,
+        tickInterval: 50,
+        accessibility: {
+            description: 'Months of the year'
+        }
+    },
+    tooltip: {
+        valuePrefix: '$',
+        headerFormat: '<b>{point.x}</b> Days Until Election<br/>',
+        shared: true,
+        distance: 30,
+        Padding: 5
+    },
+    plotOptions: {
+        series: {
+            lineWidth: 4,
+            states: {
+                hover: {
+                    lineWidth: 5
+                }
+            }
+            //animation: {
+            //    duration: 5000
+            //}
```

Comment:
Delete this part.

Reply:
oh question here, I wanted to do this animation effect to show the pacing being together. However i realized it probably will start the animation of the page load. Is there a way to start the animation when the chart comes into view? Found something like [this](https://stackoverflow.com/questions/59307440/animate-highcharts-series-when-chart-appears-in-view-when-using-data-module) but not sure how this would be implemented with the iframe situation. If its a pain i can just drop this idea too

Reply:
i just took this out for now but still curious about using this idea in the future possibly